Source/UINavigation/Private/UINavBlueprintFunctionLibrary.cpp

```diff
@@ -6,6 +6,11 @@
 #include "GameFramework/InputSettings.h"
 #include "UINavSettings.h"
 #include "UINavComponent.h"
+#include "Kismet/GameplayStatics.h"
+#include "IXRTrackingSystem.h"
+
+FString Platform = UGameplayStatics::GetPlatformName();
+FString HMD = GEngine->XRSystem->GetSystemName().ToString();
```

Comment:
I don't think this is a very safe thing to do actually. Sorry for flipping back on this but this should probably go back to the RespectsRestriction function

Reply:
Fixed